sdk/nodejs/runtime/invoke.ts

```diff
@@ -483,20 +486,67 @@ export function call<T>(
                     }
                 }
 
-                // If the value the engine handed back is or contains an unknown value, the resolver will mark its value as
-                // unknown automatically, so we just pass true for isKnown here. Note that unknown values will only be
-                // present during previews (i.e. isDryRun() will be true).
-                resolver(<any>result, true, containsSecrets, deps);
-            } catch (e) {
-                resolver(<any>undefined, true, false, undefined, e);
+                return {
+                    result: result,
+                    containsSecrets: containsSecrets,
+                    dependencies: deps,
+                    isKnown: true,
+                };
             } finally {
                 done();
             }
         }),
         label,
     );
+}
+
+/**
+ * Call is the corresponding function to invokeOutput.
```

Comment:
I think rather than us tying call to invoke, we should just document what call is -- as mentioned above I reckon we can just take the existing doc string and use that here, and have a tweaked version for `callSingle` that explains how we'll just pluck a value out of the object.